src/test/js/spec/AdminReportSpec.js

```diff
@@ -0,0 +1,48 @@
+describe("ManagementUnitReportSelectorViewModel Spec", function () {
+    beforeAll(function() {
+        window.fcConfig = {
+            imageLocation:'/'
+        };
+        if (!window.bootbox) {
+            window.bootbox = {alert:function(){}}
+        }
+        // These are defined in the ecodata-client-plugin which doesn't package javascript as a node module so
+        // getting them for testing purposes is a bit messy, so they are just being stubbed.
+        ko.simpleDirtyFlag  = function() {
+            this.isDirty = ko.observable(false);
+            return this;
+        };
+    });
+    afterAll(function() {
+        delete window.fcConfig;
+    });
+
+    it("Should able to download the report", function() {
+        var options = {generateMUReportInPeriodUrl:'/test/url'};
+        var model = new ManagementUnitReportSelectorViewModel(options);
```

Comment:
This is the approach I suggested above - passing the parameters to the view model constructor function.